workflows/csharp/sdk/order-processor/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Dapr.Client;
+using Dapr.Workflow;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using WorkflowConsoleApp.Activities;
+using WorkflowConsoleApp.Models;
+using WorkflowConsoleApp.Workflows;
+
+const string storeName = "statestore";
+
+// The workflow host is a background service that connects to the sidecar over gRPC
+var builder = Host.CreateDefaultBuilder(args).ConfigureServices(services =>
+{
+    services.AddDaprWorkflow(options =>
+    {
+        // Note that it's also possible to register a lambda function as the workflow
+        // or activity implementation instead of a class.
+        options.RegisterWorkflow<OrderProcessingWorkflow>();
+
+        // These are the activities that get invoked by the workflow(s).
+        options.RegisterActivity<NotifyActivity>();
+        options.RegisterActivity<ReserveInventoryActivity>();
+        options.RegisterActivity<ProcessPaymentActivity>();
+        options.RegisterActivity<UpdateInventoryActivity>();
+    });
+});
+
+// Start the app - this is the point where we connect to the Dapr sidecar
+using var host = builder.Build();
+host.Start();
+
+using var daprClient = new DaprClientBuilder().Build();
+
+// Wait for the sidecar to become available
+while (!await daprClient.CheckHealthAsync())
+{
+    Thread.Sleep(TimeSpan.FromSeconds(5));
+}
+
+// Wait one more second for the workflow engine to finish initializing.
+// This is just to make the log output look a little nicer.
+Thread.Sleep(TimeSpan.FromSeconds(1));
```

Comment:
People are going to assume this is needed in their code, and it seems arbitrary to have this 1s wait. Can this be removed?

Reply:
This was useful mainly in the interactive console example. Since this version is non-interactive, I think it's fine to remove this sleep.

Reply:
The sleep makes the log cleaner so it's my vote to keep it included. I believe that the comment should provide enough information to the user that the sleep isn't needed in production or their own personal environment.